lib/github_config.rb

```diff
@@ -10,7 +10,7 @@ def initialize(application_id:, application_secret:, restrict_access_to_organiza
   def self.from_values(config_hash)
     new(application_id:                   config_hash["github_application"]["application_id"],
         application_secret:               config_hash["github_application"]["application_secret"],
-        restrict_access_to_organizations: config_hash["restrict_access_to_organizations"].split(','))
+        restrict_access_to_organizations: config_hash["restrict_access_to_organizations"]&.split(','))
```

Comment:
What is the purpose of this change?

Reply:
`config_hash["restrict_access_to_organizations"]` is populated by the ENV variable `ALLOWED_ORGANIZATIONS`.  
This is the logic for determining access:

```
def organization_permitted?(organizations)
    if @restrict_access_to_organizations
      !(@restrict_access_to_organizations & organizations).empty?
    else
      true
    end
  end
```

So in order the machine test github user who is not in any org, the valueof `@restrict_access_to_organizations` needs to be falsy.  But it was impossible to get a falsy value that wouldn't throw an error when executing `config_hash["restrict_access_to_organizations"].split(','))`.  I spent a lot of time trying to find an ENV variable format that would parse and set it the way I wanted, but I had no choice but to change the code to make this work.

Reply:
I see, that sounds reasonable.